Recorder/insurees-flow-recorder.jmx

```diff
@@ -0,0 +1,3578 @@
+            <boolProp name="HTTPSampler.postBodyRaw">true</boolProp>
+            <elementProp name="HTTPsampler.Arguments" elementType="Arguments">
+              <collectionProp name="Arguments.arguments">
+                <elementProp name="" elementType="HTTPArgument" enabled="true">
+                  <boolProp name="HTTPArgument.always_encode">false</boolProp>
+                  <stringProp name="Argument.value">{&quot;operationName&quot;:null,&quot;query&quot;:&quot;mutation {\n      createInsuree(\n        input: {\n          clientMutationId: \&quot;9463b13c-b09f-4ba8-8b9e-c7bfd8900fab\&quot;\n          clientMutationLabel: \&quot;Create insuree - testchfid\&quot;\n          \n          chfId: \&quot;testchfid\&quot;\n    lastName: \&quot;test\&quot;\n    otherNames: \&quot;test\&quot;\n    genderId: \&quot;F\&quot;\n    dob: \&quot;2023-10-25\&quot;\n    head: false\n    \n    \n    \n    \n    \n    \n    \n    cardIssued:false\n    \n    \n    \n    familyId: 35\n    \n    status: \&quot;AC\&quot;\n    \n    \n    \n    jsonExt: \&quot;{}\&quot;\n        }\n      ) {\n        clientMutationId\n        internalId\n      }\n    }&quot;}</stringProp>
+                  <stringProp name="Argument.metadata">=</stringProp>
+                  <boolProp name="HTTPArgument.use_equals">true</boolProp>
+                </elementProp>
+              </collectionProp>
+            </elementProp>
+            <stringProp name="HTTPSampler.domain">${SERVER_NAME}</stringProp>
+            <stringProp name="HTTPSampler.port">${SERVER_PORT}</stringProp>
+            <stringProp name="HTTPSampler.protocol">http</stringProp>
+            <stringProp name="HTTPSampler.contentEncoding">UTF-8</stringProp>
+            <stringProp name="HTTPSampler.path">/api/graphql</stringProp>
+            <stringProp name="HTTPSampler.method">POST</stringProp>
+            <boolProp name="HTTPSampler.follow_redirects">true</boolProp>
+            <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
+            <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
+            <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
+            <boolProp name="HTTPSampler.BROWSER_COMPATIBLE_MULTIPART">false</boolProp>
+            <stringProp name="GraphQLHTTPSampler.operationName"></stringProp>
+            <stringProp name="GraphQLHTTPSampler.query">
+    mutation {
+      createInsuree(
+        input: {
+          clientMutationId: &quot;9463b13c-b09f-4ba8-8b9e-c7bfd8900fab&quot;
+          clientMutationLabel: &quot;Create insuree - testchfid&quot;
+          
+          chfId: &quot;testchfid&quot;
+    lastName: &quot;test&quot;
+    otherNames: &quot;test&quot;
+    genderId: &quot;F&quot;
+    dob: &quot;2023-10-25&quot;
+    head: false
+    
+    
+    
+    
+    
+    
+    
+    cardIssued:false
+    
+    
+    
+    familyId: 35
```

Comment:
family ID and location can be different under different databases. We have think how to parametrize this one.